.gitignore

```diff
@@ -5,3 +5,4 @@ artifacts/
 .terraform/
 .terraform.d
 vendor/
+build.sh
```

Comment:
I think `build.sh` should not be included in the `.gitignore` as it's not autogenerated during the build/test steps